docs/writing_tests.rst

```diff
@@ -160,7 +160,11 @@ Multiple workflows can use the same custom test like this:
 points to the folder where the named workflow was executed. This allows writing
 of advanced python tests for each file produced by the workflow.
 
+Custom tests must follow the `conventions for Python test discovery
+<https://docs.pytest.org/en/7.2.x/explanation/goodpractices.html#conventions-for-python-test-discovery>`_,
+which constrains the names of files and functions containing custom tests.
+
 .. note::
 
     stdout and stderr are available as files in the root of the
-    ``workflow_dir`` as ``log.out`` and ``log.err`` respectively.
+    ``workflow_dir`` as ``log.out`` and ``log.err`` respectively.
```

Comment:
I presume this is a ``git`` error as the line has not changed? Or am I missing something?

Reply:
The end of file new line was unintentionally deleted. I have put it back.